requirements.txt

```diff
@@ -11,6 +11,7 @@ django-cors-headers==2.1.0
 django-filter==1.1.0
 djangorestframework==3.7.7
 djangorestframework-jwt==1.11.0
+django-rest-multiple-models==2.0.1
```

Comment:
what is this package? @prashant2796 @Par010

Reply:
This is the package from which I am inheriting ObjectMultipleModelAPIView in my global_search views .
It allows me to write multiple queryset with its serializer in a single Querylist.
So this way I dont have to make a seperate serializer for my global_search view.

Reply:
but the point of this activity was to make you create serializer to handle polymorphic data. Time and again I have been stressing the importance of *not* using packages unless absolutely necessary. There is not learning if you just plug in a package. Same was for tags in the Notes project last sem and this sem the amount of time we invested in social auth. Let it be for now, but I had explicitly told you to create a custom serializer which would take data agnostic to the model. @abhn I want you to express your views wrt packages vs development philosophy in the current use case.

Reply:
Yes! Absolutely. Even I started off implementation with creating custom serializer and readonlymodelviewset. But then I was really lost halfway through. Was having various issues and conditions related to it which I also wanted to discuss. Then I realized this search feature was taking way more time then it should. Then I moved to package solution.

Reply:
I think it is fine, as long as you understand what the package is doing internally, or what it might be doing, and use a package to save some time. That's how things usually work.

But if you're using that package as a black box, then make it a point to understand how it works whenever you get time. It might work now, but this development paradigm is an open invitation for hard to find/fix bugs.

Reply:
Absolutely! Learning is always the first priority.Thanx for the feedback :)